zephyr/module.yml

```diff
@@ -15,26 +15,26 @@ blobs:
     description: "Firmware for NXP IW612 Wi-Fi + Bluetooth single-chip solution"
     doc-url: https://www.nxp.com/products/wireless-connectivity/wi-fi-plus-bluetooth-plus-802-15-4/2-4-5-ghz-dual-band-1x1-wi-fi-6-802-11ax-plus-bluetooth-5-4-plus-802-15-4-tri-radio-solution:IW612
   - path: rw61x/rw61x_sb_ble_15d4_combo_a2.bin
-    sha256: 41e9837790c23c25b5e29defef87cbcc559f452c3bbcc4c894429237e3a2e98d
+    sha256: 7d94d41c4778b691292a325b881e972cb676467e57bfc804b0d6f8926d5a8dda
     type: img
     version: '1.0'
     license-path: zephyr/blobs/license/LA_OPT_NXP_Software_License.txt
-    url: https://github.com/nxp-mcuxpresso/mcux-sdk/raw/MCUX_2.15.000_RW610_K32W1/components/conn_fwloader/fw_bin/rw61x_sb_ble_15d4_combo_a2.bin
+    url: https://github.com/NXP/wifi_nb_fw/raw/nxp-v3.7.0/rw61x_sb_ble_15d4_combo_a2.bin
     description: "BLE and 15.4 combo firmware for RW612 A2 boards"
-    doc-url: https://github.com/nxp-mcuxpresso/mcux-sdk/tree/main/components/conn_fwloader/readme.txt
+    doc-url: https://github.com/NXP/wifi_nb_fw/blob/wifi_nb_fw_rel/readme.txt
   - path: rw61x/rw61x_sb_ble_a2.bin
-    sha256: 983f8a9e04305ae2ca44d501e469ab2144781e23b249813e373ec9969417b74d
+    sha256: 7db0dcfa76d69dae95296ab6aff84fa444a34876adccd7e83c2afb8faaeb5b1a
     type: img
     version: '1.0'
     license-path: zephyr/blobs/license/LA_OPT_NXP_Software_License.txt
-    url: https://github.com/nxp-mcuxpresso/mcux-sdk/raw/MCUX_2.15.000_RW610_K32W1/components/conn_fwloader/fw_bin/rw61x_sb_ble_a2.bin
+    url: https://github.com/NXP/wifi_nb_fw/raw/nxp-v3.7.0/rw61x_sb_ble_a2.bin
     description: "BLE firmware for RW612 A2 boards"
-    doc-url: https://github.com/nxp-mcuxpresso/mcux-sdk/tree/main/components/conn_fwloader/readme.txt
+    doc-url: https://github.com/NXP/wifi_nb_fw/blob/wifi_nb_fw_rel/readme.txt
```

Comment:
The doc-url need to point to the same version that the blob URL is or these will get out of sync

Reply:
thanks, please review again